packages/compat/README.md

```diff
@@ -77,15 +76,15 @@ import { fixupPluginRules } from "@eslint/compat";
 import somePlugin from "eslint-plugin-some-plugin";
 
 export default [
-    {
-        plugins: {
-            // insert the fixed plugin instead of the original
-            somePlugin: fixupPluginRules(somePlugin)
-        },
-        rules: {
-            "somePlugin/rule-name": "error"
-        }
-    }
+	{
```

Comment:
prettier uses tabs by default? Seems like too much space from the existing one?

Reply:
Yes, tabs are default in Prettier.

This is only 1 tab, but GitHub by default renders tabs as 8 spaces so it looks huge.

Reply:
> GitHub by default renders tabs as 8 spaces

It's configurable:

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/managing-your-tab-size-rendering-preference

Reply:
Actually, prettier's default indentation is 2 spaces. But we configured it in this project to use tabs:

https://github.com/eslint/rewrite/blob/3e9eb67964327ef908ab27fa0f14990580ec801c/prettier.config.js#L2